Makefile

```diff
@@ -2904,6 +2931,46 @@ distclean: clean toolsclean
 dist:
 	git archive --format zip --output $(CLIENTBIN)-$(VERSION).zip HEAD
 
+#############################################################################
+# INSTALL (only for Linux platforms)
+#############################################################################
+
+# Shell scripts for running binaries
+
+$(B)/$(CLIENTBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(CLIENTBIN) "$$@"' >> $@
+
+$(B)/$(SERVERBINSH): 
+	@echo '#!/usr/bin/env sh' > $@
+	@echo 'cd $(COPYBINDIR)' >> $@
+	@echo './$(SERVERBIN) "$$@"' >> $@
+
+install: release
+ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu"))
+	$(echo_cmd) "Installing for Linux platform in $(COPYBINDIR) and $(PREFIX)"
+	@$(MKDIR) -p $(COPYBINDIR)
+	@cd $(BR) && for file in $(NAKED_TARGETS); do \
+		$(INSTALL) -D $$file $(COPYBINDIR)/$$file; \
+	done
+#	@(cd $(BR) && $(INSTALL) $() $(COPYBINDIR) )
+# TODO: create these .sh files:
+	@$(INSTALL) -d $(PREFIX)/bin
+	@$(INSTALL) -D -m755 $(BR)/$(CLIENTBINSH) $(BR)/$(SERVERBINSH) $(PREFIX)/bin
+# Install the .desktop, icon files, license, etc.
+	@$(INSTALL) -D -m644 "misc/io.github.grangerhub.Tremulous.png" "$(PREFIX)/share/pixmaps/tremulous.png"
+	@$(INSTALL) -D -m644 "misc/io.github.grangerhub.Tremulous.desktop" \
+			 "$(PREFIX)/share/applications/tremulous.desktop"
```

Comment:
No, this is incorrect. The whole point of renaming the desktop file (and its matching launchable tag in the AppStream metadata file) was to keep it at the new `io.github.grangerhub.Tremulous.desktop` name. Installing it as `tremulous.desktop` makes no sense at this point and will break things for AppStream metadata.

If you want to install the desktop/icon files as just `tremulous.*`, rename the original files in the misc directory and also change the launchable tag in AppStream file to just "tremulous.desktop" (but don't change the actual id and metainfo file name - that should always stay in rDNS format).

Reply:
Got it. I renamed them to Tremulous-Grangerhub.

Reply:
No, this is still incorrect and even more broken now! I was talking about the rDNS formatted name and I even sent suggestions to apply.

Reply:
By the way, rDNS naming conventions for desktop files will probably became compulsory on many Linux distributions / packaging formats in the future. So my change was also future-proof.

If you use this "Tremulous-Grangerhub" name format because of the upcoming official Grangerhub Snap, then please consider doing these changes in downstream (patch for the Snap package), not directly in upstream.

But it is up to you. I just do not personally like these changes, that's probably my problem.